delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/multifetch/model/MultiFetchResponse.scala

```diff
@@ -0,0 +1,118 @@
+package ch.epfl.bluebrain.nexus.delta.sdk.multifetch.model
+
+import cats.data.NonEmptyList
+import ch.epfl.bluebrain.nexus.delta.rdf.RdfError
+import ch.epfl.bluebrain.nexus.delta.rdf.Vocabulary.contexts
+import ch.epfl.bluebrain.nexus.delta.rdf.jsonld.api.{JsonLdApi, JsonLdJavaApi}
+import ch.epfl.bluebrain.nexus.delta.rdf.jsonld.context.{ContextValue, RemoteContextResolution}
+import ch.epfl.bluebrain.nexus.delta.rdf.jsonld.encoder.JsonLdEncoder
+import ch.epfl.bluebrain.nexus.delta.rdf.syntax.jsonLdEncoderSyntax
+import ch.epfl.bluebrain.nexus.delta.sdk.jsonld.JsonLdContent
+import ch.epfl.bluebrain.nexus.delta.sdk.model.ResourceRepresentation.{CompactedJsonLd, Dot, ExpandedJsonLd, NQuads, NTriples, SourceJson}
+import ch.epfl.bluebrain.nexus.delta.sdk.model.{BaseUri, ResourceRepresentation}
+import ch.epfl.bluebrain.nexus.delta.sdk.multifetch.model.MultiFetchResponse.Result
+import ch.epfl.bluebrain.nexus.delta.sdk.multifetch.model.MultiFetchResponse.Result.itemEncoder
+import ch.epfl.bluebrain.nexus.delta.sourcing.model.{ProjectRef, ResourceRef}
+import io.circe.syntax.EncoderOps
+import io.circe.{Encoder, Json, JsonObject}
+import monix.bio.{IO, UIO}
+
+/**
+  * A response for a multi-fetch operation
+  * @param format
+  *   the formats in which the resource should be represented
+  * @param resources
+  *   the result for each resource
+  */
+final case class MultiFetchResponse(format: ResourceRepresentation, resources: NonEmptyList[Result]) {
+
+  /**
+    * Encode the response as a Json payload
+    */
+  def asJson(implicit base: BaseUri, rcr: RemoteContextResolution): UIO[Json] = {
+    val encodeItem = itemEncoder(format)
+    resources.traverse(encodeItem).map { r =>
+      Json.obj(
+        "format"    -> format.asJson,
```

Comment:
If the caller specifies the format, I wonder if it's useful to return it

Reply:
Adding it does not hurt and as users of the API often gets confused between the formats, it can help to answer their questions

Reply:
I think it hurts in the sense that it's code to maintain and which we test, but if you think it's worth it that's fine